galeracluster/source/documentation/scriptable-sst.rst

```diff
@@ -103,6 +103,12 @@ These parameters are passed to all state transfer scripts, regardless of method
 
    ``--defaults-file`` The script is given the path to the ``my.cnf`` configuration file.
 
+   ``--total N`` This progress reporting parameter indicates the new SST stage and reports the estimated total work.
+
+   ``--complete N`` This progress reporting parameter reports the work completed so far.
```

Comment:
Same as above. These are not script parameters, these are script feedback to the caller.